### PR TITLE
TM-86: Endpoint /vocab and /vocab/:id

### PR DESCRIPTION
**Pull request by ManuelTS**
Also brings an insomnia.config.json, please use it and make sure to always merge with master first to not override anything, closes #6

### CODE REVIEW

**Review by ja-fra — CHANGES_REQUESTED** [review]
Please wait for #9 to be merged and align to the service-structure there:

* Service-Layer (VocabService, PersistenceService) should not have dependencies on the API-Layer (express)
* API-Layer does the conversion between API-Models (OpenAPI-Spec) and Database-Models

File: be/src/controlers/rest.router.ts [hunk 1]
```diff
@@ -1,77 +1,121 @@
...
-    entityCount: -1
-  };
-}
-
-router.get('/vocab', async (req: Request, res: Response) => {
-  res.json({statusCode:200});
-});
+router.get('/vocab', (req: Request, res: Response) => processVocab(req, res));
+
+router.get('/vocab/:id', (req: Request, res: Response) => processVocab(req, res, true));
```
Comment (ja-fra) [inline be/src/controlers/rest.router.ts:33]: `GET /vocab` and `GET /vocab/:id` do **not** have the same response format.

The first is a list (with paging info, etc), the second is a single entity.

**Review by ja-fra — CHANGES_REQUESTED** [review]
Please try to keep your changes local: do not touch/cleanup/rewrite every function within one PR as this makes merging and parallel work difficult.

File: be/src/controlers/rest.router.ts (hunk 1 above)
Comment (ManuelTS) [reply be/src/controlers/rest.router.ts:33]: Bugfix accomplished, please conduct a validation. 

Empty request on empty DB or with non existing ID return now:
Screenshot from 2021-12-13 10-46-00
Screenshot from 2021-12-13 10-46-12

Same with some DB entries and GET of a list and read of a single vocab:
Screenshot from 2021-12-13 10-46-35
Screenshot from 2021-12-13 10-46-46

**Pull request closed without merging**